components/meta-packages/history/history

```diff
@@ -975,7 +975,10 @@ release/openindiana-welcome@0.5.11,5.11-0.148 release/os-welcome@0.5.11-0.151.1.
 release/registration@0.5.11,5.11-2018.0.0.0
 runtime/java/runtime64@0.5.11,5.11-2017.0.0.0 runtime/java/openjdk8@1.8.112,5.11-2016.1.0.0
 runtime/java@0.5.11,5.11-2017.0.0.0 runtime/java/openjdk8@1.8.112,5.11-2016.1.0.0
+runtime/nodejs@0.12.18-2020.0.1.3 runtime/nodejs-10@10.20.1-2020.0.1.0
```

Comment:
Wrong copy/paste for the multiple nodejs-10?

Reply:
From what I understand the "runtime/nodejs-10@10.20.1-2020.0.1.0" part is in order to let installed nodejs-0.12.18 packages be replaced by the runtime/nodejs-10@10.20.1-2020.0.1.0 package.
Likewise for nodejs-6 and nodejs-8. Am I wrong?

Reply:
Sorry I misread it.